pkg/plugin/plugin.go

```diff
@@ -657,3 +691,22 @@ func CmdCheck(args *skel.CmdArgs) error {
 	log.Print("CHECK is not yet implemented, pretending everything is fine")
 	return nil
 }
+
+func loadConfFromCache(cRef string) (*cachedNetConf, error) {
+	netCache := &cachedNetConf{}
+	netConfBytes, err := utils.ReadCache(cRef)
+	if err != nil {
+		return nil, fmt.Errorf("error reading cached NetConf with name %s: %v", cRef, err)
+	}
+
+	if err = json.Unmarshal(netConfBytes, netCache); err != nil {
+		return nil, fmt.Errorf("failed to parse NetConf: %v", err)
+	}
+
+	return netCache, nil
+}
+
+// getCRef unique identifier for a container interface
+func getCRef(cid, podIfName string) string {
```

Comment:
Could we move this to the cache module since it has no use for anything but cache?

Reply:
ok, now added two more modules config.go, types.go to avoid circular dependency and more cohesive code.  with this change, these 2 methods are moved into config.go as that file is appropriate for this.
is that ok ?